color.py

```diff
@@ -0,0 +1,9 @@
+import enum
+
+
+class Color(enum.Enum):
```

Comment:
For easier usage you can give the values of the enum to more meaning full value like : 

`
class Color(enum.Enum):
   Red = 'red'
    Blue = 'blue'
    Green = 'green'
    Yellow = 'yello'
`